ReSharper.FSharp/src/FSharp/FSharp.Psi.Daemon/FSharp.Psi.Daemon.fsproj

```diff
@@ -13,8 +13,6 @@
     <TargetFramework>net472</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="PsiFeaturesNamespaceZoneMarker.fs" />
-    <Compile Include="PsiNamespaceZoneMarker.fs" />
```

Comment:
Are these changes needed in this PR?

Reply:
I touched fsproj around this place and decided to bring the position of neighboring files to a more expected state. We can also do this in a separate PR.